specification/cognitiveservices/data-plane/ContentModerator/v1.0/ContentModerator.json

```diff
@@ -1816,42 +2286,286 @@
+        "parameters": [
+          {
+            "name": "Content-Type",
+            "description": "The content type.",
+            "required": true,
+            "type": "string",
+            "in": "header"
+          },
+          {
+            "name": "teamName",
+            "description": "Your team name.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "reviewId",
+            "description": "Id of the review.",
+            "in": "path",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "timescale",
+            "description": "Timescale of the video .",
+            "in": "query",
+            "required": false,
+            "type": "number"
+          },
+
+          {
+            "name": "frameImageZip",
+            "description": "Zip file containing frame images.",
+            "in": "formData",
+            "required": true,
+            "type": "file"
+
           },
           {
-            "$ref": "#/parameters/contentTypeHeader"
+            "name": "frameMetadata",
+            "description": "Metadata of the frame.",
+            "in": "formData",
+            "required": true,
+            "type": "string"
+
           },
           {
-            "$ref": "#/parameters/ImageStreamParameter"
+            "$ref": "#/parameters/ocpApimSubscriptionKeyParameter"
           }
         ],
         "consumes": [
-          "image/gif",
-          "image/jpeg",
-          "image/png",
-          "image/bmp",
-          "image/tiff"
+          "multipart/form-data"
         ],
         "responses": {
-          "200": {
-            "description": "OK",
+          "204": {
+            "description": "No Content"
+          },
+          "default": {
+            "description": "Error response.",
             "schema": {
-              "$ref": "#/definitions/AddImage"
+              "$ref": "#/definitions/APIError"
             }
           }
         },
-        "x-ms-examples": {
-          "Add Image request": {
-            "$ref": "./examples/AddImageStreamResource.JSON"
-          }
-        },
+       
         "produces": [
           "application/json"
         ]
+
       }
     }
+    
   },
   "definitions": {
+    "Frames": {
+      "description": "The response for a Get Frames request.",
+      "properties": {
+        "reviewId": {
+          "description": "Id of the review.",
+          "type": "string"
+        },
+        "videoFrames": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/Frame"
+          }
+        }
+      }
+    },
+    "Frame": {
+      "description": "Video frame property details.",
+      "properties": {
+        "timestamp": {
+          "description": "Timestamp of the frame.",
+          "type": "string"
```

Comment:
Shoudl this be date-time?

Reply:
Timestamp of the frame in our case is "Tics/sec".

Reply:
So maybe "integer"?

Reply:
I have changed this to integer.